core/src/test/scala/unit/kafka/network/SocketServerTest.scala

```diff
@@ -94,6 +93,7 @@ class SocketServerTest {
 
   @BeforeEach
   def setUp(): Unit = {
+    server.enableRequestProcessing(Map.empty).get(1, TimeUnit.MINUTES)
```

Comment:
`enableRequestProcessing` will start network thread to accept connection. Start it in `BeforeEach`, so that it won't be initialized unexpectedly.